src/components/Home/Home.js

```diff
@@ -0,0 +1,40 @@
+import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import Stats from '../Stats/Stats';
+import Coin from '../Coin/Coin';
+import './home.css';
+
+export const CoinsList = ({ coins }) => {
+  const coinsList = coins.map((coin) => <Coin coin={coin} key={Math.random()} />);
+  return (
+    <ul className="crypto-list">
+      {coinsList}
+    </ul>
+  );
```

Comment:
Your design
![image](https://github.com/fezzopro/react-capstone-project/assets/78251927/c28a1621-35b1-4e32-8a63-d809f9fed472)

Template
![image](https://github.com/fezzopro/react-capstone-project/assets/78251927/dd7227f3-3d92-46f7-b6ec-261fead4336a)

The items here are too jam-packed, I suggest we add more padding `top` and `bottom` to improve readability

Reply:
Thank you for pointing this out, I have improved the UI in order to improve readability